addons/survey/models/survey_user.py

```diff
@@ -289,24 +317,21 @@ def _prepare_statistics(self):
         }) for user_input in self)
 
         scored_questions = self.mapped('predefined_question_ids').filtered(
-            lambda question: question.question_type in ['simple_choice', 'multiple_choice']
+            lambda question: question.question_type in ['simple_choice', 'multiple_choice'] or question.is_scored_question
         )
 
         for question in scored_questions:
-            question_answer_correct = question.suggested_answer_ids.filtered(lambda answer: answer.is_correct)
+            if question.question_type in ['simple_choice', 'multiple_choice']:
+                question_answer_correct = question.suggested_answer_ids.filtered(lambda answer: answer.is_correct)
+                # avoid counting multi-choice questions with no right answer
+                if not question_answer_correct:
+                    continue
```

Comment:
What does that mean exactly ?
Is that the "fix" you mention in your latest comment on this PR ?

Reply:
Yes, this essentially makes sure we don't count multi-choice questions with no correct answer(s) towards the final pie chart performance statistics shown at the end of a certification quiz for the survey taker.

Reply:
I'm sorry but I' not entirely sure what you mean.
Can you give me a complete use case of the behavior with and without the fix with some screenshots maybe?
Will make it easier to understand for me.

Also, if this is a problematic behavior, maybe we should extract the fix for an earlier stable version, but I'll need to understand the problem first.

Reply:
No worries, the simplest case is setting up a new certification survey with only 1 multiple choice question with no `is_correct` choices. Then take and submit the survey without selecting any choices:

![simplesurvey3](https://user-images.githubusercontent.com/57942143/72517136-2278bb00-3853-11ea-9b52-6d2c3a0e577e.png)

In the end it counts that question as correct (screenshot is from my branch with all other changes except for this check. Latest master runbot will show it as both 'Unanswered' and 'Correct'):

![simplesurvey4](https://user-images.githubusercontent.com/57942143/72517189-315f6d80-3853-11ea-8dd7-a2b6e27ba1b0.png)

This change makes it so an unanswered choice question which has no correct answer(s) isn't counted as a question related to the pie chart:
![simplesurvey5](https://user-images.githubusercontent.com/57942143/72518075-cd3da900-3854-11ea-9d12-c559c192b96c.png)

All other cases where there is at least 1 correct answer are still properly counted. It is overall a very small thing, but incorrect behavior in my opinion. If you agree that it should be a fix then I can split it into a separate commit. The double counting of a multi-choice question with no answer in this pie chart appears in SAAS 13.1 and 13. Doesn't look like this pie chart exists in SAAS 12.3 and earlier.

Reply:
Ok thanks for the full explanation, now I get it.
Then yes the fix will have to be split in a separate commit and put in a 13.0 branch.

I think you can create another small task concerning that issue.
So we can move on with this one while we discuss the issue on a separate task and validate with SBU.

Reply:
Small task created. Removed this 'fix' from this branch.